packages/ui/components/document/document-download-button.tsx

```diff
@@ -24,44 +23,18 @@ export const DocumentDownloadButton = ({
   disabled,
   ...props
 }: DownloadButtonProps) => {
-  const { toast } = useToast();
-
   const [isLoading, setIsLoading] = useState(false);
 
   const onDownloadClick = async () => {
-    try {
-      setIsLoading(true);
-
-      if (!documentData) {
-        return;
-      }
-
-      const bytes = await getFile(documentData);
-
-      const blob = new Blob([bytes], {
-        type: 'application/pdf',
-      });
-
-      const link = window.document.createElement('a');
-      const baseTitle = fileName?.includes('.pdf') ? fileName.split('.pdf')[0] : fileName;
+    setIsLoading(true);
```

Comment:
It looks like this is never set to false in the scenario that `documentData` is not available since `finally` no longer exists

You should be able to just shift this below the if statement.

Reply:
👍